README.md

```diff
@@ -65,7 +65,9 @@ docker-compose build --no-cache
 ```
 
 ## Developer Instructions
-Developers can quickly get started by setting up the dev environment using the instructions above. The database is seeded with the following admin account.
+Developers can quickly get started by setting up the dev environment using the instructions above. It is necessary to execute 
+'bundle exec rake db:seed' to populate the sample data and access admin account.
```

Comment:
It's not necessary to seed the db for starting development, how does the following look? 

> To seed the database with some sample data, run `bundle exec rake db:seed`. The admin credentials after seeding will be as follows ...

Reply:
It seems good. I'll make these changes :)

Reply:
I think you already made the changes. Right? @ankitkataria